doc/CMakeLists.txt

```diff
@@ -18,6 +18,7 @@ set(LATEXMKOPTS "-halt-on-error -no-shell-escape" CACHE STRING "Default latexmk
 set(DT_TURBO_MODE OFF CACHE BOOL "Enable DT turbo mode")
 set(DOC_TAG "development" CACHE STRING "Documentation tag")
 set(DTS_ROOTS "${ZEPHYR_BASE}" CACHE STRING "DT bindings root folders")
+set(BUILD_MANPAGES OFF CACHE BOOL "Ask doxygen to also build man pages")
```

Comment:
can you prefix this option with DOXYGEN since this is only for doxygen? Sphinx also supports man pages.